### PR TITLE
test

### PR DESCRIPTION
**Pull request by fahmizainal17**
test again

### CODE REVIEW

**Comment by fahmizainal17** [comment]
hi aidil please check and merge if necessary

File: app/1_Pre_EDA.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from module.EDA_BE import display_eda_info, round_decimal_columns, check_and_fill_null_values, check_and_remove_duplicates
 
 # Insert Tittle and markdown
-st.title("Hi this Fahmi Exploratory Data Analysis🕵️")
+st.title("Hi this Fahmi again Exploratory Data Analysis🕵️")
```
Comment (aidiladhamc7) [inline app/1_Pre_EDA.py]: alright great for testing , now i have put the ruleset into the main branch , i will accept for this time for testing , next , we will see more for improvement of the code

**Review by aidiladhamc7 — CHANGES_REQUESTED** [review]

File: app/1_Pre_EDA.py (hunk 1 above)
Comment (fahmizainal17) [reply app/1_Pre_EDA.py]: done change

**Review by fahmizainal17 — COMMENTED** [review]
cannot approve

**Review by aidiladhamc7 — APPROVED** [review]

**Pull request merged**